{"uid":"4b4757e66a1912dae1a509f688f20b0f","name":"categories","children":[{"name":"Product defects","children":[{"name":"assert 2 == 1\n +  where 2 = len(<snappi.snappi.MacsecMetricIter object at 0x7ff32d6db180>)\n +    where <snappi.snappi.MacsecMetricIter object at 0x7ff32d6db180> = <snappi.snappi.MetricsResponse object at 0x7ff32cb5b8d0>.macsec_metrics","children":[{"name":"test_encrypt_with_mka_multiple_devices","uid":"953cd81f8226152e","parentUid":"ef332ea001ff09f1df2f30b5ed7129ed","status":"failed","time":{"start":1747341697732,"stop":1747341812237,"duration":114505},"flaky":true,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]}],"uid":"ef332ea001ff09f1df2f30b5ed7129ed"},{"name":"Exception: Time out occurred while waiting for stats to be accumulated","children":[{"name":"test_bidir_tagged_flows","uid":"3cd5bc57e4c8dee4","parentUid":"5a7c9ad8292efb3b06ec20e1edee3646","status":"failed","time":{"start":1747341697732,"stop":1747341772516,"duration":74784},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]}],"uid":"5a7c9ad8292efb3b06ec20e1edee3646"},{"name":"snappi_ixnetwork.exceptions.SnappiIxnException:   File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/snappi_ixnetwork/snappi_api.py\", line 452, in set_control_state\n    self.traffic_item.transmit(request_payload)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/snappi_ixnetwork/trafficitem.py\", line 1484, in transmit\n    self._api._traffic.Apply()\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/testplatform/sessions/ixnetwork/traffic/traffic.py\", line 1395, in Apply\n    return self._execute(\"apply\", payload=payload, response_object=None)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/base.py\", line 452, in _execute\n    response = self._connection._execute(url, payload)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/connection.py\", line 293, in _execute\n    return self._send_recv(\"POST\", url, payload)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/connection.py\", line 576, in _send_recv\n    return self._poll()\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/connection.py\", line 644, in _poll\n    return self._process_response_status_code(\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/connection.py\", line 480, in _process_response_status_code\n    raise BadRequestError(message, response.status_code)\n  Error in L2/L3 Traffic Apply","children":[{"name":"test_vxlan_b2b_v6","uid":"5ea63a23d56b44ae","parentUid":"258fd5130fb77a243675638ef508c901","status":"failed","time":{"start":1747341697732,"stop":1747341849622,"duration":151890},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]}],"uid":"258fd5130fb77a243675638ef508c901"},{"name":"Exception: Time out occurred while waiting for stats to be as expected","children":[{"name":"test_mka_stats","uid":"40f9cc2e4a346668","parentUid":"3596725c47ed555d65da3fbf3423e1cd","status":"failed","time":{"start":1747341697732,"stop":1747341850499,"duration":152767},"flaky":true,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]}],"uid":"3596725c47ed555d65da3fbf3423e1cd"}],"uid":"8fb3a91ba5aaf9de24cc8a92edc82b5d"}]}